docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -136,6 +138,8 @@ network:
   port: 26500
   minKeepAliveInterval: 30s
   maxMessageSize: 4MB
+  socketReceiveBuffer: 4MB
+  socketSendBuffer: 4MB
```

Comment:
:x: The default value is supposed to be 1MB.

Reply:
Same reasoning as in the first comment.